megamek/src/megamek/server/Server.java

```diff
@@ -15268,17 +15268,22 @@ private void removeDeadAttacks() {
     /**
      * Apply damage to mech for zweihandering (melee attack with both hands) as per
      * pg. 82, CamOps
-     * @param ae - the attacking entity
-     * @param missed - did the attack missed. If so PSR is necessary.
-     */
-    private void applyZweihanderSelfDamage(Entity ae, boolean missed) {
+     * 
+     * @param ae           - the attacking entity
+     * @param missed       - did the attack missed. If so PSR is necessary.
+     * @param criticalLocs - the locations for possible criticals, should be one or
+     *                     both arms depending on if it was an unarmed attack (both
+     *                     arms) or a weapon attack (the arm with the weapon).
+     */
+    private void applyZweihanderSelfDamage(Entity ae, boolean missed, List<Integer> criticalLocs) {
```

Comment:
I think this would be "easier" to use if `criticalLocs` was varargs, as in `Integer... criticalLocs`

Not at all a requirement.